ci/script.sh

```diff
@@ -1,32 +1,15 @@
 set -ex
 
 main() {
-    # root
-    cargo build --verbose
-    cargo test
-
-    # grpc
-    pushd grpc
-    cargo build --verbose
-    cargo test
-    popd
-
     # web viewer - client
     pushd web_viewer
     pushd client
     npm run build
     popd
 
-    # web viewer - server
-    cargo build --verbose
-    cargo test
-    popd
-
-    # sdl viewer
-    pushd sdl_viewer
-    cargo build --verbose
-    cargo test
-    popd
+    # root
```

Comment:
this is not the root directory but web_viewer. add another popd to get to root

Reply:
done!